--- ffmpeg.old/configure	2025-07-22 11:05:50.318431772 +0800
+++ ffmpeg/configure	2025-07-22 10:58:33.299941680 +0800
@@ -5915,8 +5915,12 @@
         enable dos_paths
         ;;
     linux)
+        disable symver
         enable section_data_rel_ro
         enabled_any arm aarch64 && enable_weak linux_perf
+        SLIB_INSTALL_NAME='$(SLIBNAME)'
+        SLIB_INSTALL_LINKS=
+        SHFLAGS='-shared -Wl,-soname,$(SLIBNAME)'
         ;;
     irix*)
         target_os=irix
